app/graph/types/source_type.rb

```diff
@@ -27,24 +26,6 @@
     }
   end
 
-  connection :project_sources, -> { ProjectSourceType.connection_type } do
-    resolve ->(source, _args, _ctx) {
-      source.project_sources
-    }
-  end
-
-  connection :projects, -> { ProjectType.connection_type } do
-    resolve ->(source, _args, _ctx) {
-      source.projects
-    }
-  end
-
-  connection :medias, -> { ProjectMediaType.connection_type } do
```

Comment:
I think we still want to know which medias have been published by a source.

Reply:
Ok will keep it